parser/include/parser/pluginhandler.h

```diff
@@ -36,6 +36,12 @@ class PluginHandler
    */
   void loadPlugins(std::vector<std::string>& skipParserList_);
 
+  /**
+   * Return the list of loaded plugins based the file names in the dynamic
+   * libraries.
```

Comment:
While the data member behind this functionality is named `_dynamicLibraries`, it is a private data member, so this method description can be confusing for someone just reading the public API.  
I think it should simply state that this methods returns the loaded and not skipped plugin names.

Also, an *"on"* preposition is missing after the word *"based"*.